pom.xml

```diff
@@ -39,6 +39,7 @@
         <spring-cloud.version>Edgware.SR5</spring-cloud.version>
         <spring-cloud-services.version>1.6.3.RELEASE</spring-cloud-services.version>
         <spring-cloud-sso.version>1.1.0.RELEASE</spring-cloud-sso.version>
+        <spring-cloud-cloudfoundry-connector>2.0.3.RELEASE</spring-cloud-cloudfoundry-connector>
```

Comment:
You don't really think it's a bad idea to use libs which are based on framework5 while we're on framework4?

Reply:
So that's then even wrong even in `spring-cloud-dataflow-server-cloudfoundry`.